404.html

```diff
@@ -15,12 +15,12 @@
 <!--Fallback page, don't allow indexing-->
 <meta name="robots" content="noindex">
 <h1>An open source P2P digital currency</h1>
+{% comment %}
 <p style="font-size:125%;">Bitcoin is a currency, a protocol, and a software that enables
 <ul style="font-size:125%">
-  <li>Instant peer to peer transactions</li>
```

Comment:
>Instant peer to peer transactions

I don't have the permissions to make edits, so I'm commenting here. I don't think this line is 100% false, it's just not good enough. 

I suggest:

>Directly verifiable digital cash payments - fast confirmation.